src/DataGrid.tsx

```diff
@@ -141,10 +146,6 @@ export interface DataGridProps<R, SR = unknown, K extends Key = Key> extends Sha
   sortColumns?: Maybe<readonly SortColumn[]>;
   onSortColumnsChange?: Maybe<(sortColumns: SortColumn[]) => void>;
   defaultColumnOptions?: Maybe<DefaultColumnOptions<R, SR>>;
-  groupBy?: Maybe<readonly string[]>;
```

Comment:
Most of the grouping logic has been removed from `DataGrid` to `TreeDataGrid`. The only thing remining is row focus logic